src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -183,6 +182,19 @@ protected override Conversion GetInterpolatedStringConversion(BoundExpression so
 
             Debug.Assert(elementType is { });
 
+            if (collectionTypeKind == CollectionExpressionTypeKind.ImplementsIEnumerable)
```

Comment:
>if [](http://example.com/codeflow?start=12&length=2)

Need to update breaking change document with the additional conversion requirements: an accessible .ctor callable with no arguments, and an accessible Add method callable with an argument of the iteration type.